### ISSUE TITLE
Text output is not escaped

### ISSUE THREAD

**Issue by louisabraham** [iodide-project/iodide#824]
In OCaml, a typical interaction is

``` ocaml
# let square x = x * x;;
val square : int -> int = <fun>
```

The `<fun>` creates a html tag in the output when returning a string from an evaluator or calling `iodide.output.text` with https://iodide-project.github.io/dist/iodide.pyodide-20180623.js

This was just how I discovered it, but printing things like `<b>bold</b>` also works.

A simple fix is to use the innerText attribute (see comment below).

**Comment by mdboom** [comment]
Thanks for pointing out the additional bug.  I'll tack on a fix to #823.

I think we should definitely use `innerText` in `iodide.output.text`.  For creating HTML DOM nodes, we already have `iodide.output.element` which provides a safer way to create DOM nodes.

**Comment by louisabraham** [comment]
I also think the style should imitate a <pre> block:

``` css
  display: block;
  unicode-bidi: embed;
  font-family: monospace;
  white-space: pre;
```

**Comment by bcolloran** [comment]
@mdboom is this fixed as of #823, or still more to do?
